packages/ros2/resources/convert__msg.cpp.em

```diff
@@ -34,12 +34,15 @@ conversion_dependency = 'soss/rosidl/ros2/{}/msg/convert__msg__{}.hpp'.format(
 // Include the Node API so we can subscribe and advertise
 #include <rclcpp/node.hpp>
 
+// Include export macros in order to export symbols in windows
+#include <soss/soss_export.hpp>
+
 namespace soss {
 namespace ros2 {
 namespace @(namespace_variable) {
 
 //==============================================================================
-class Subscription final
+class SYSTEM_HANDLE_EXPORT Subscription final
```

Comment:
Are you sure that these macros are needed for dynamically loaded libraries? I may be wrong, but I thought they were only necessary for creating the IMPLIB. But maybe I'm confusing myself with symbol visibility in Linux.

Reply:
You are right. They are not necessary. Thanks @mxgrey!